recipe/meta.yaml

```diff
@@ -3,8 +3,8 @@
 {% set variant = os.environ.get('BLAS_VARIANT', 'openblas') %}
 
 # The version should be in the form X.Y, where X denotes major infrastructural change to how blas is managed,
-# and Y denoting blas priority. Currently Y=0 not default, Y=1 default blas.
-{% set Y = 1 if variant == 'openblas' else 0 %}
+# and Y denoting blas priority. Currently Y=0 not default, Y=2 default blas.
+{% set Y = 2 if variant == 'openblas' else 0 %}
 package:
   name: blas
   version: 1.{{ Y }}
```

Comment:
We need to bump the major version if we are changing the variants that are available (and there ordering).